test/graphql/types/Fund/createdAt.test.ts

```
@@ -0,0 +1,255 @@
+  role: string;
+  organizationMembershipsWhereMember: Array<{
+    role: string;
+    organizationId: string;
+  }>;
+};
+
+describe("Fund CreatedAt Resolver Tests", () => {
+  let ctx: GraphQLContext;
+  let mockFund: FundType;
+  let mocks: ReturnType<typeof createMockGraphQLContext>["mocks"];
+  let createdAtResolver: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)